.travis.yml

```diff
@@ -21,9 +21,9 @@ cache:
 before_install:
   - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then openssl aes-256-cbc -K $encrypted_8d91199f8619_key -iv $encrypted_8d91199f8619_iv -in secring.gpg.enc -out secring.gpg -d; fi
 install:
-  - rvm use 2.3.0 --install --fuzzy
+  - rvm use 2.6.5 --install --fuzzy
   - gem install sass
```

Comment:
```suggestion
```

This can be removed as Jekyll 4 includes its own sass parser internally.

Reply:
Thanks!!